upstream/CVE-2022-3294.json

```diff
@@ -0,0 +1,113 @@
+{
+  "id": "CVE-2022-3294",
+  "modified": "2022-11-08T21:33:26Z",
+  "published": "2022-11-08T21:33:26Z",
+  "summary": "Node address isn't always verified when proxying",
+  "details": "Users may have access to secure endpoints in the control plane network. Kubernetes clusters are only affected if an untrusted user can modify Node objects and send proxy requests to them. Kubernetes supports node proxying, which allows clients of kube-apiserver to access endpoints of a Kubelet to establish connections to Pods, retrieve container logs, and more. While Kubernetes already validates the proxying address for Nodes, a bug in kube-apiserver made it possible to bypass this validation. Bypassing this validation could allow authenticated requests destined for Nodes to to the API server's private network.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:H/PR:H/UI:N/S:U/C:H/I:H/A:H"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.25.0"
+            },
+            {
+              "last_affected": "1.25.3"
```

Comment:
```suggestion
              "fixed": "1.25.4"
```

```
➜  kubernetes git:(master) git tag --contains 3ce7f86bdbbcd0c4f4f734e24e5798cb764c3f08 | sort -V | head -n 5
v1.25.4
v1.25.5
v1.25.5-rc.0
v1.25.6
v1.25.6-rc.0
```

Reply:
Updated in https://github.com/aquasecurity/vuln-list-k8s/pull/90/commits/063bdeb04c2c2db8bc490e0055c67317e18f5d35